config.lua.dist

```diff
@@ -67,7 +67,7 @@ timeBetweenExActions = 1000
 
 -- Map
 -- NOTE: set mapName WITHOUT .otbm at the end
-mapName = "forgotten"
+mapName = "atlas"
```

Comment:
I don't think it's a good idea to rename the map file, we don't own it, so keep it like before